app/components/primer/component.rb

```diff
@@ -144,5 +144,12 @@ def should_raise_error?
     def should_raise_aria_error?
       !Rails.env.production? && raise_on_invalid_aria? && !ENV["PRIMER_WARNINGS_DISABLED"]
     end
+
+    def trimmed_content(trim: false)
+      return content unless trim && content.present?
+
+       # strip unsets `html_safe`, so we have to set it back again to guarantee that HTML blocks won't break
+      content.html_safe? ? content.strip.html_safe : content.strip # rubocop:disable Rails/OutputSafety
+    end
```

Comment:
IMO it's a little strange to have a method named `trimmed_content` that doesn't do the thing it's directly responsible for doing if you pass a special flag. What if we conditionally call this method instead?

```suggestion
    def trimmed_content
      return content unless content.present?

       # strip unsets `html_safe`, so we have to set it back again to guarantee that HTML blocks won't break
      content.html_safe? ? content.strip.html_safe : content.strip # rubocop:disable Rails/OutputSafety
    end
```

Reply:
Good catch 👍 I changed it as suggested.